.gitignore

```diff
@@ -2,3 +2,4 @@ build/
 node_modules/
 .idea
 *.iml
+package-lock.json
```

Comment:
I guess we should commit this file

Reply:
i don't think we need it though. What do you say @toarm ?

Reply:
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
I know they recommend doing so, however we haven't been doing this consistently, neither have we used the feature afaik. @toarm should decide

Reply:
Honestly I never thought about package-lock.json. For me it feels initially wrong to push such a file but on the other side they are recommending it x/

I'd like to test it. If it results in merge conflicts and other annoying stuff we can remove it. Acutally I dont see any benefit in pushing it but lets stick to the recommendations for a while and see if it works out...

Reply:
@toarm @KrzysztofKozlowskiOBSS re-added package-lock.json